alpine-openjdk/README.md

```diff
@@ -0,0 +1,102 @@
+# alpine-openjdk
+
+A Docker image for running [OpenJDK][openjdk], based on Alpine Linux.
+This image belongs to a suite of images [documented here][dockeralpine].
+
+## Features
+
+This image features:
+
+- [Alpine Linux][alpinelinux]
+- [s6][s6] and [s6-overlay][s6overlay]
+- [OpenJDK][openjdk]
+
+## Versions
+
+- `8.92.14`, `latest` [(Dockerfile)](https://github.com/smebberson/docker-alpine/blob/alpine-openjdk-v8.92.14/alpine-openjdk/Dockerfile)
```

Comment:
Are you wanting to version these images based on the JDK version? I haven't really versioned any other images that way, simply because if I change a parent image, the version numbers get out of sync really easily and therefore I thought, what is the point...

Have a read of, https://github.com/smebberson/docker-alpine#versioning. Based on that. I'd suggest these images start at `1.0.0`. What are your thoughts?